tsconfig.json

```diff
@@ -13,6 +13,7 @@
       "./typings"
     ],
 
+    "resolveJsonModule": true,
```

Comment:
this is for our messages files?

Reply:
> this is for our messages files?

yeah, no reason to keep tsc from not seeing them